libs/navigation/website/src/lib/footer/__snapshots__/footer-container.spec.tsx.snap

```diff
@@ -24,7 +24,23 @@ exports[`Footer Container should render Default 1`] = `
       </svg>
```

Comment:
navbar and footer are implemented in another PR, that's why here there are only tests?

Reply:
It's because I fixed the mock, so they showed up :D